crates/rolldown_plugin_replace/src/plugin.rs

```diff
@@ -114,6 +122,15 @@ impl Plugin for ReplacePlugin {
     if self.try_replace(args.code, &mut magic_string)? {
       return Ok(Some(HookTransformOutput {
         code: Some(magic_string.to_string()),
+        map: if self.sourcemap {
+          Some(magic_string.source_map(SourceMapOptions {
```

Comment:
We can't pass the sourcemap of `string_wizard` directly to rolldown. This will cause the same problem in https://github.com/rolldown/rolldown/pull/1426.

We need to first think about how to deal with this issue.

Reply:
Ok, I'll finish `object_guards` first